src/CodeGeneration/ApiGenerator/RestSpecification/XPack/slm.get_status.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
I'm sure planned but create a PR for this to elastsearch as well.

Reply:
Yup, was going to add to the client first then open one in ES

Reply:
Opened in https://github.com/elastic/elasticsearch/pull/49759